src/content/toys.ts

```diff
@@ -9,34 +9,54 @@
  */
 export function jack_in() {
     // chinese characters - taken from the unicode charset
-    const chinese = "田由甲申甴电甶男甸甹町画甼甽甾甿畀畁畂畃畄畅畆畇畈畉畊畋界畍畎畏畐畑".split(
-        "",
-    )
+    const chinese = "田由甲申甴电甶男甸甹町画甼甽甾甿畀畁畂畃畄畅畆畇畈畉畊畋界畍畎畏畐畑".split("")
     const colour = "#0F0" // green text
-    rain(chinese, colour)
+    rain(makeBlock(), chinese, colour)
 }
 
-export function no_mouse() {
-    rain([" "], "#FFF", 0) // No characters, unused colour code, no darkening
+
+export function music() {
+    // music characters - taken from the unicode charset
+    const music = "𝄞𝄟𝄰𝅘𝅥𝅮𝅘𝅥𝅯𝅘𝅥𝅰𝄽".split("")
+    const colour = "#ead115"
+    rain(makeBlock(), music, colour)
 }
 
-export const snow = () => rain(["❄"], "#FFF", 0.15)
+export function no_mouse() {
+    makeBlock()
+}
 
-export function rain(characters: string[], colour, darkening = 0.05) {
+function makeBlock() {
     const d = document.createElement("div")
+    d.className = "_tridactyl_no_mouse_"
     d.style.position = "fixed"
     d.style.display = "block"
-    d.style.width = "100%"
-    d.style.height = "100%"
+    d.style.width = String(window.innerWidth)
+    d.style.height = String(window.innerHeight)
     d.style.top = "0"
     d.style.left = "0"
     d.style.right = "0"
     d.style.bottom = "0"
     d.style.zIndex = "1000"
     d.style.opacity = "0.5"
+    document.body.appendChild(d)
+    return d
+}
+
+export function removeBlock() {
+    Array.from(document.getElementsByClassName("_tridactyl_no_mouse_")).map(el=>{
+        if((el as any).intid!=null){
+            clearInterval((el as any).intid)
```

Comment:
Something like this, maybe?
```suggestion
    Array.from(document.getElementsByClassName("_tridactyl_no_mouse_")).map((el: Element & { intid?: number | null}) => {
        if(typeof el.intid === "number") {
            clearInterval(el.intid)
```